src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { PropsWithChildren } from 'react';
+import * as S from './Stepper.style';
+
+export type StepperProps = PropsWithChildren<any>;
+
+const Stepper: React.FC<StepperProps> = ({ children, ...rest }: StepperProps) => (
+	<S.Stepper>
+		{React.Children.map(children, (child, index) => [
+			<div className="line" />,
```

Comment:
Move this on top of the loop

```suggestion
			<div className="stepper__progress-bar" />,
```